localization/ndt_scan_matcher/src/util_func.cpp

```diff
@@ -254,3 +254,9 @@ std::vector<geometry_msgs::msg::Pose> createRandomPoseArray(
 
   return poses;
 }
+
+double norm(const geometry_msgs::msg::Point & p1, const geometry_msgs::msg::Point & p2)
+{
+  return std::sqrt(
+    std::pow(p1.x - p2.x, 2.0) + std::pow(p1.y - p2.y, 2.0) + std::pow(p1.z - p2.z, 2.0));
```

Comment:
I think 

```cpp
const double dx = (p1.x - p2.x);
const double dy = (p1.y - p2.y);
const double dz = (p1.z - p2.z);
std::sqrt(dx * dx + dy * dy + dz * dz);
```

Is faster than calling pow

Reply:
Thanks!
Fixed in https://github.com/autowarefoundation/autoware.universe/pull/1757/commits/caf72fd686fa1f2ca43ebfe07b85a8ed8c8970fc